openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -91,11 +93,11 @@ def create_jwt_token_dict(token_dict, oauth_adapter, use_asymmetric_key=None):
     )
 
     jwt_token_dict = token_dict.copy()
-    # Note: only "scope" is not overwritten at this point.
```

Comment:
If it is true that we are now overwriting all response values, we no longer need to copy the dict. We could just have:
```
jwt_token_dict = {
    ...
}
```

Reply:
Some additional thoughts:
1. Can you restore the comment, and mention the refresh_token?
2. I’m wondering if it would be best if the refresh_token had the extra scope as well, and if we didn’t need to add it here, because it would have been added earlier?
3. If we called the api with the three password grant scopes explicitly (as a test from the command line), does it work or complain about the user_id request? If it complains, does that indicate where the scope update should happen?

Reply:
I think #2 would be more accurate/consistent, because the refresh token would match the actual scopes. 
That said, I agree that this shouldn't be a blocker, so I added a potential code comment just to capture what is going on.